Monika-After-Story/MonikaModDev
# First time crash greet improvements

Improves/fixes a few things in the crash greet. 

- Currently you can quit by force closing the game at any time during this greet and there is no warning and upon next load, you get the greeting for a subsequent crash. We will now disable quit once you get past the first menu since she is sure you are there then. We also won't set `persistent._mas_crashed_before ` to True until this same point, so if you do quit before this, you still get this greet next load
- The transition from the black screen to the spaceroom with the lights on wasn't great. No matter what, we showed an empty spaceroom with no desk, and then she just dissolves in, which didn't really make sense. Now if you turn the light on, she's just there when the lights turn on, and if she turns the lights on, we start with empty desk, and then she sits down.
- We've finally added an actual light switch sfx, handling a longstanding `TODO`. File from https://www.freesoundslibrary.com/light-switch-sound-effect/ (note to @ThePotatoGuy to add a list of our free sounds)
- Added APIs for setting the quit dialogue box messages
    - `mas_setQuitMsg` - sets the quit text, yes message, and no message
    - `mas_resetQuitMsg` - resets the quit text, yes message, and no message to the affection-based values

## Testing

- verify force closing the game after the initial menu now functions as you would expect and results in an appropriate greeting upon next load
- verify force closing the game before the menu results in getting this same greet upon next load
- verify transition from the black screen is appropriate and light switch sound is good
